ibm/service/power/resource_ibm_pi_network_port_attach.go

```diff
@@ -25,40 +25,63 @@ func ResourceIBMPINetworkPortAttach() *schema.Resource {
 
 		CreateContext: resourceIBMPINetworkPortAttachCreate,
 		ReadContext:   resourceIBMPINetworkPortAttachRead,
-		UpdateContext: resourceIBMPINetworkPortAttachUpdate,
 		DeleteContext: resourceIBMPINetworkPortAttachDelete,
 		Importer:      &schema.ResourceImporter{},
 
 		Timeouts: &schema.ResourceTimeout{
 			Create: schema.DefaultTimeout(60 * time.Minute),
 			Delete: schema.DefaultTimeout(60 * time.Minute),
-			Update: schema.DefaultTimeout(60 * time.Minute),
 		},
 		Schema: map[string]*schema.Schema{
-			"port_id": {
-				Type:     schema.TypeString,
```

Comment:
The required argument can't be removed directly needs to follow Terraform guidelines 
https://www.terraform.io/plugin/sdkv2/best-practices/deprecations

Reply:
Is this  resources we are exposin to the users for the first time ?

Reply:
this resource was present in an earlier version but without documentation, the network-port resource will be deprecated and the features of network-port will combine in the network-port-attach resource